src/hu/elte/txtuml/stdlib/util/HashMap/HashmapImplementation.java

```diff
@@ -0,0 +1,92 @@
+package hu.elte.txtuml.stdlib.util.HashMap;
+
+import java.util.HashMap;
+import java.util.Set;
+import hu.elte.txtuml.api.model.Collection;
+
+public class HashmapImplementation implements Hashmap{

```

Comment:
Rename this class `HashMapImplementation` so it matches the name of the interface.